common/utils/src/main/resources/error/error-classes.json

```
@@ -1958,6 +1974,11 @@
       "The join condition <joinCondition> has the invalid type <conditionType>, expected \"BOOLEAN\"."
     ]
   },
+  "KRYO_BUFFER_OVERFLOW" : {
+    "message" : [
+      "Kryo serialization failed: <exceptionMsg>. To avoid this, increase <bufferSizeConfKey> value."
```

Does `exceptionMsg` bring additional info if we already know that a buffer overflow?
```suggestion
      "Kryo serialization failed: <exceptionMsg>. To avoid this, increase \"<bufferSizeConfKey>\" value."
```